include/openssl/ct.h

```diff
@@ -398,7 +402,7 @@ int SCT_LIST_validate(const STACK_OF(SCT) *scts, CT_POLICY_EVAL_CTX *ctx);
  * Returns < 0 on error, >= 0 indicating bytes written (or would have been)
  * on success.
  */
-int i2o_SCT_LIST(const STACK_OF(SCT) *a, unsigned char **pp);
+__owur int i2o_SCT_LIST(const STACK_OF(SCT) *a, unsigned char **pp);
```

Comment:
`o2i` currently only exists for EC. I'm not a fan of adding new cryptic APIs but let's say the 'o' is for 'opaque'...

That doesn't change the fact that `o2i_SCT_signature` doesn't return a signature, but rather, sets it on the SCT. So, it should be a setter (resp. getter). We can leave the other ones as they are, for lack of time and better ideas.